# RNA-Seq_course

## Concepts and Main Aspects of RNA-Seq
-----------------------------------

- 25th, 28th and 29th April 2016
- Wellcome Trust Centre for Human Genetics, Seminar Rooms A/B


**Main organizers**

- Irina Pulyakhina
- Helen Lockstone


**Teachers**

- Paolo Piazza
- Helen Lockstone
- Irina Pulyakhina
- Davis McCarthy
- ...


**Lectures**

You can find lectures given during the course here:

- [Introduction to NGS](...)
- [Experimental design](...)
- [Introduction to RNA-Seq](https://github.com/jknightlab/RNA-Seq_course/blob/master/2016-04-25_RNAseqCourse_RNASeq-QC_irina.pdf)
- [Alignment](https://github.com/jknightlab/RNA-Seq_course/blob/master/2016-04-25_RNAseqCourse_alignment_irina.pdf)
- [Transcript quantification](...)
- [DE analysis and edgeR](...)
- [Pathway analysis](...)
- [Visualisation](https://github.com/jknightlab/RNA-Seq_course/blob/master/2016-04-25_RNAseqCourse_vizualise_irina.pdf)
- [Splicing](https://github.com/jknightlab/RNA-Seq_course/blob/master/2016-04-29_RNAseqCourse_splicing_irina.pdf)
- [Cutting edge technologies](...)
- [Single cell sequencing](...)


**Data**

Where to find the VMs:

- [fow Windows users](http://www.well.ox.ac.uk/Portable-VirtualBox.exe)
- [for MacOS users](http://www.well.ox.ac.uk/VirtualBox.zip)

Instructions on how to install the VM can be found
[here](https://github.com/jknightlab/RNA-Seq_course/blob/master/vm_instructions.pdf).

UCSC session for the second practical on visualisation can be found
[here](https://genome-euro.ucsc.edu/cgi-bin/hgTracks?db=hg19&lastVirtModeType=default&lastVirtModeExtraState=&virtModeType=default&virtMode=0&nonVirtPosition=&position=chr22%3A20904012-20924513&hgsid=213561077_dEH3EOWETpcUrPqKhZwX7vWMsZYB).

*for GMS students*

To get sequencing data that you generated during the NGS course in March,
please contact Irina Pulyakhina (room 00-038, irina@well.ox.ac.uk).

-------------
Designed by Irina Pulyakhina, irina@well.ox.ac.uk

